source/catalog.html

```diff
@@ -0,0 +1,326 @@
+              <td class="product-card__table-data">500 &#8381;</td>
+            </tr>
+          </table>
+
+          <button class="primary-button product-card__button" type="button">Заказать</button>
+        </li>
+
+        <li class="product-card">
+          <span>Показать еще 100500 товаров</span>
+          <span>На самом деле вкусов гораздо больше!</span>
+          <button class="product-card__button" type="button">Показать все</button>
+        </li>
+      </ul>
+
+      <div class="extra-product-catalog">
+        <h2 class="secondary-title-catalog">Дополнительные товары</h2>
+        <ul>
+          <li class="extra-product-card">
+            <span class="extra-product-card__title">Сахарозаменитель</span>
+            <table class="extra-product-card__table">
+              <tr class="extra-product-card__table-row">
+                <td class="extra-product-card__table-data">1 упаковка (100 г)</td>
+                <td class="extra-product-card__table-data">200 &#8381;</td>
+              </tr>
+            </table>
+            <button class="primary-button extra-product-card__button" type="button">Заказать</button>
+          </li>
+
+          <li class="extra-product-card">
+            <span class="extra-product-card__title">Питьевая вода</span>
+            <table class="extra-product-card__table">
+              <tr class="extra-product-card__table-row">
+                <td class="extra-product-card__table-data">5 литров</td>
+                <td class="extra-product-card__table-data">50 &#8381;</td>
+              </tr>
+            </table>
+            <button class="primary-button extra-product-card__button" type="button">Заказать</button>
+          </li>
+
+          <li class="extra-product-card">
+            <span class="extra-product-card__title">Молоко</span>
+            <table class="extra-product-card__table">
+              <tr class="extra-product-card__table-row">
+                <td class="extra-product-card__table-data">1 литр</td>
+                <td class="extra-product-card__table-data">100 &#8381;</td>
+              </tr>
+            </table>
+            <button class="primary-button extra-product-card__button" type="button">Заказать</button>
+          </li>
+
+          <li class="extra-product-card">
+            <span class="extra-product-card__title">Витамины</span>
+            <table class="extra-product-card__table">
+              <tr class="extra-product-card__table-row">
+                <td class="extra-product-card__table-data">1 упаковка (30 г)</td>
+                <td class="extra-product-card__table-data">300 &#8381;</td>
+              </tr>
+            </table>
+            <button class="primary-button extra-product-card__button" type="button">Заказать</button>
+          </li>
+        </ul>
+
+        <div class="present">
+          <span class="present__text">Закажите все и получите чехол для кота в подарок!</span>
+        </div>
```

Comment:
Опять же, хочется поменьше элементов и посемантичнее:

```suggestion
        <p class="present">
          Закажите все и получите чехол для кота в подарок!
        </p>
```

Reply:
👍